logstash-core/src/main/java/org/logstash/instrument/metrics/UptimeMetric.java

```diff
@@ -63,7 +63,7 @@ public UptimeMetric(final String name) {
         this(name, System::nanoTime);
     }
 
-    UptimeMetric(final String name, final LongSupplier nanoTimeSupplier) {
+    public UptimeMetric(final String name, final LongSupplier nanoTimeSupplier) {
```

Comment:
Can remain package-private; I had initially implemented the new flow-metrics in a separate package, but that made testing much more complex.

```suggestion
    UptimeMetric(final String name, final LongSupplier nanoTimeSupplier) {
```